.github/workflows/update-gov-zone-file.yml

```
@@ -27,4 +27,5 @@ jobs:
           git add gov.txt
           timestamp=$(date -u)
           git commit -m "CZDS Update ${timestamp}" || exit 0
+          git pull
           git push
```

I think this will do the same thing, without adding a merge commit to the `git` history:
```suggestion
          git fetch
          git rebase
          git add gov.txt
          timestamp=$(date -u)
          git commit -m "CZDS Update ${timestamp}" || exit 0
          git push
```

Just a suggestion.

---

Thank you for the suggestion! Brought the option up with the team and there was a preference for keeping commit messages given that this the case a simultaneous merge conflict occurs is rare. Appreciate the comment on rebase and will keep it in mind for future projects that use it over merge to pull changes!